Description: Don't fail the ppd test if the cups filters is not already installed in the correct path
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Last-Update: 2014-01-06

--- a/ppd/Makefile.am
+++ b/ppd/Makefile.am
@@ -51,4 +51,4 @@
 testppd_SOURCES=
 
 testppd:
-	cupstestppd $(dist_cupsmodel_DATA)
+	cupstestppd -W filters $(dist_cupsmodel_DATA)
